server/app.js

```diff
@@ -4,26 +4,28 @@ const path = require("path");
 const cookieParser = require("cookie-parser");
 const logger = require("morgan");
 const config = require("config");
-// const bodyParser = require("body-parser");
+const bodyParser = require("body-parser");
+const multer = require('multer');
+
 
 const userRouter = require("./routes/userRoute");
 const loginRouter = require("./routes/loginRoute");
 
-
-
 if (!config.get("jwtKey")) {
   console.error('FATAL ERROR:PRIVATE KEY NOT SET!')
   process.exit(1)
 }
 
 const app = express();
 
-// app.use(bodyParser.json({ extended: true }));
-// app.use(bodyParser.urlencoded({ extended: true }));
+app.use(bodyParser.json({ extended: true, limit: '50mb' }));
+app.use(bodyParser.urlencoded({ limit: '50mb', extended: true, parameterLimit: 50000 }));
+app.use(multer({ dest: './public/uploads/' }).single('file'));
```

Comment:
This line is what was causing the problem. Just remove it and it should work. It was overriding the config, so the image was being uploaded locally instead